static/app/components/events/breadcrumbs/combinedBreadcrumbsAndLogsSection.tsx

```diff
@@ -0,0 +1,61 @@
+import {Fragment} from 'react';
+
+import type {BreadcrumbsDataSectionProps} from 'sentry/components/events/breadcrumbs/breadcrumbsDataSection';
+import BreadcrumbsDataSection from 'sentry/components/events/breadcrumbs/breadcrumbsDataSection';
+import useOrganization from 'sentry/utils/useOrganization';
+import {LogsPageParamsProvider} from 'sentry/views/explore/contexts/logs/logsPageParams';
+import {LogsIssuesSection} from 'sentry/views/explore/logs/logsIssuesSection';
+import {useExploreLogsTable} from 'sentry/views/explore/logs/useLogsQuery';
+
+/**
+ * This component is a coordinator and provider wrapper to determine which section to display, collapse etc. since only one section should be displayed by default.
+ */
+export function CombinedBreadcrumbsAndLogsSection({
```

Comment:
Do they need to be combined since they both render their own section? Or are they sharing something.

Reply:
This coordinator is for following the ask in discuss-issues
> If logs team can figure out a way to collapse breadcrumbs and expand logs sections automatically when an error has logs we'll try that

It's fetching data before rendering both so that the collapse can be passed. If that's not a requirement we can have them as separate sections.

Ideally we only see logs expanded if logs exist and not breadcrumbs as they'll eventually replace breadcrumbs.